project/apps/collaboration-service/src/domain/ports/collaboration.service.ts

```diff
@@ -42,23 +42,20 @@ export class CollaborationService {
   }
 
   /**
-   * Retrieves the active collaboration for a given user.
+   * Retrieves the active collaborations for a given user.
    *
-   * @param userId - The ID of the user whose active collaboration is to be retrieved.
-   * @returns A promise that resolves to the active collaboration data transfer object (CollabDto).
+   * @param userId - The ID of the user whose active collaborations is to be retrieved.
+   * @returns A promise that resolves to the active collaboration data transfer objects (CollabDto[]).
    * @throws Will handle and log any errors that occur during the retrieval process.
    */
-  async getActiveCollab(userId: string): Promise<CollabDto> {
+  async getActiveCollabs(userId: string): Promise<CollabDto[]> {
     try {
-      const collabs = await this.collabRepository.findActive(userId);
+      const activeCollabs = await this.collabRepository.findActive(userId);
 
       // we are assuming only one active collaboration per user for now
```

Comment:
nit: remove this comment

Reply:
also help to rerun `pnpm --filter dtos gen:types-and-enums` please, i added a new 'document' column 😄